conda-recipes/meta.yaml

```diff
@@ -27,22 +27,33 @@ requirements:
     - vtk=8.2
     - eigen
     - tbb-devel=2020
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ openmp }} # [osx]
 
   build:
     - cmake>=3.14
     - swig=3
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - {{ openmp }} # [osx]
 
   run:
-    - python 
+    - python
+    - numpy=1.18
+    - vtk=8.2
     - {{ pin_compatible('numpy') }}
     - {{ pin_compatible('vtk') }}
     - scipy
     - pandas
     - jinja2
     - deprecated
     - psutil
+    - cc3dext_maboss
```

Comment:
This can't be here, since it will create a dependency loop. Furthermore, if I understand correctly, then the extensions are optional features. 

Technically, it's ok for now. But once the ``cc3d`` conda package is published, it will become a dependency of ``cc3dext_maboss``. Proposing to remove it in https://github.com/CompuCell3D/CompuCell3D/pull/509.